NumeRe.cbp

```diff
@@ -228,6 +229,12 @@
 		<Unit filename="externals/dtl/dtl/dtl.hpp" />
 		<Unit filename="externals/dtl/dtl/functors.hpp" />
 		<Unit filename="externals/dtl/dtl/variables.hpp" />
+		<Unit filename="externals/stduuid/catch/catch.hpp" />
+		<Unit filename="externals/stduuid/catch/catch_reporter_automake.hpp" />
+		<Unit filename="externals/stduuid/catch/catch_reporter_sonarqube.hpp" />
+		<Unit filename="externals/stduuid/catch/catch_reporter_tap.hpp" />
+		<Unit filename="externals/stduuid/catch/catch_reporter_teamcity.hpp" />
```

Comment:
Please evaluate, whether those includes are actually necessary (I do not think so).

Reply:
> GSL was not necessary in the end? Or is it included automatically?

Adding the stduuid directory is enough

Nope, catch is not needed!